cmd/landscaper-gardenlet/app/landscaper-gardenlet.go

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	gardenletconfig "github.com/gardener/gardener/pkg/gardenlet/apis/config"
+	gardenletconfigv1alpha1 "github.com/gardener/gardener/pkg/gardenlet/apis/config/v1alpha1"
+	"github.com/gardener/gardener/pkg/landscaper/gardenlet"
+	"github.com/gardener/gardener/pkg/landscaper/gardenlet/apis/imports"
+	importsv1alpha1 "github.com/gardener/gardener/pkg/landscaper/gardenlet/apis/imports/v1alpha1"
+	importvalidation "github.com/gardener/gardener/pkg/landscaper/gardenlet/apis/imports/validation"
+	"github.com/gardener/gardener/pkg/version/verflag"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+// NewCommandStartLandscaperGardenelet creates a *cobra.Command object with default parameters
+func NewCommandStartLandscaperGardenelet(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "landscaper-gardenlet",
+		Short: "Launch the Gardener landscaper component for the Gardenlet.",
+		Long:  "Gardener landscaper component for the Gardenlet. Sets up the Garden cluster and deploys the Gardenlet with TLS bootstrapping to automatically register the configured Seed cluster.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			verflag.PrintAndExitIfRequested()
+
+			if len(args) != 0 {
+				return errors.New("arguments are not supported. Please only set the path to the configuration file via environment variable \"IMPORTS_PATH\"")
+			}
+
+			return run(ctx)
+		},
+	}
+
+	// add version flag
+	flags := cmd.Flags()
+	verflag.AddFlags(flags)
+	return cmd
+}
+
+func run(ctx context.Context) error {
+	var err error
+
+	landscaperOperation, importPath, componentDescriptorPath, err := getLandscaperEnvironmentVariables()
```

Comment:
```suggestion
	landscaperOperation, importPath, componentDescriptorPath, err := getLandscaperEnvironmentVariables()
```
not needed as the err is declared in line 63?